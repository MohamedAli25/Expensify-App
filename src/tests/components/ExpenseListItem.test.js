import ExpenseListItem from "../../components/ExpenseListItem";
import React from "react";
import { shallow } from "enzyme";

test("should render ExpenseListItem", () => {
  const data = {
    id: "1",
    description: "Rent",
    amount: 10000,
    createdAt: 0,
  };
  const wrapper = shallow(<ExpenseListItem {...data} />);
  expect(wrapper).toMatchSnapshot();
});
