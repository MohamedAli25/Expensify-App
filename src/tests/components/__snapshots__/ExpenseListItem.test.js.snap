// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem 1`] = `
<div>
  <Link
    to="/edit/1"
  >
    <h3>
      Rent
    </h3>
  </Link>
  <p>
    Amount: 
    10000
  </p>
  <p>
    Create At: 
    0
  </p>
</div>
`;
